common/schema/DHQauthor-TEI.xml

```diff
@@ -995,7 +999,20 @@ email (should only have an email address)
       <!-- Add @type to <title> to distinguish between article and issue titles -->
       <elementSpec ident="title" mode="change" module="core">
         <attList>
-          <attDef ident="type" mode="add">
+          <attDef ident="level" mode="add">
```

Comment:
Heh-heh. Rather than add this here, simply delete the `<attDef ident="level" mode="delete"/>`, which is the last element of this `<attList>`. (And I am chuckling as I am imagining you looked for it, did not see it, and added this in a somewhat frustrated way.)

Reply:
Fixed in 5c730290.